### PR TITLE
S3

### PR DESCRIPTION
**Pull request by creativelogistics**
I manual seeded my db kept the functionality that created 100 entries

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,35 +1,69 @@
...
+const connection = require('../database/connection.js')
 
+server.use(bodyParser());
+server.use('/api', router);
+server.use(express.static('public'));
 server.listen(port, () => console.log(`server listening on port : ${port}`));
 
-server.get('/data', (req,res) => {
+server.get('/data', (req, res) => {
+  connection.connection.query('select * from images', (err, result) => {
```
Comment (pfaithmtan) [inline server/index.js]: Why is connection written twice?

Comment (creativelogistics) [reply server/index.js]: thats the difference so the red is the past and i mad a space after comma that is after req

**Comment by creativelogistics** [comment]
Thanks yea enlist is really helpful . I did a pull by mistake . The s3 branch was meant to be a seeding feature but I ended up I ended up doing it by hand.

File: server/index.js (hunk 1 above)
Comment (pfaithmtan) [reply server/index.js]: I was referring to line 24 `connection.connection.query`, was this intentional or did you mean `connection.query`?

**Pull request merged**